app/ios/STATests/Info.plist

```diff
@@ -19,6 +19,6 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>3</string>
```

Comment:
Should this be `4` now if it should match the `versionCode` [here](https://github.com/Scottish-Tech-Army/Volunteer-app/blob/main/app/android/app/build.gradle#L132)?